### DISCUSSION TITLE
ETA of using Timber 2 in production?

### DISCUSSION THREAD

**Discussion by jaimygoetheer** [timber/timber#2666] [General]
Hi all, is there an ETA for when we can use Timber 2 in production websites?

**Comment by xsynaptic** [comment]
I've been using it in production on several WP sites for at least a year now, no issues found thus far. YMMV.

**Comment by Levdbas** [comment]
Hi @jaimygoetheer @xsynaptic , the team is hard at work to make Timber 2.0 happen. We've just created an updated roadmap with open tickets, pull requests, etc. If you would like to help out on Timber, now is the perfect timing. See https://github.com/timber/timber/issues/2741

Closing this ticket in favor of the mentioned ticket above.
